search/api.py

```diff
@@ -162,7 +167,15 @@ def search_percolate_queries(program_enrollment_id):
         django.db.models.query.QuerySet: A QuerySet of PercolateQuery matching the percolate results
     """
     conn = get_conn()
-    result = conn.percolate(get_default_alias(), USER_DOC_TYPE, id=program_enrollment_id)
+    enrollment = ProgramEnrollment.objects.get(id=program_enrollment_id)
+    doc = serialize_program_enrolled_user(enrollment)
```

Comment:
Ok, given that the code looks good, we'll just need to keep an eye our for an changes to those task usages.